index.bs

```diff
@@ -769,12 +773,28 @@ console.log(&#96;values: ${bufferE}&#96;);
 </pre>
 </div>
 
+### The {{MLNamedArrayBufferViews}} transfer algorithm ### {#mlnamedarraybufferviews-transfer}
+To <dfn for="MLNamedArrayBufferViews">transfer</dfn> an {{MLNamedArrayBufferViews}} |views|:
+1. Let |transferredViews| be a new {{MLNamedArrayBufferViews}}.
+1. For each |key| -> |value| of |views|:
+    1. Let |transferredBuffer| be the result of [=transfer array buffer|transferring=] {{ArrayBuffer}} |value|.\[[ViewedArrayBuffer]].
```

Comment:
Use https://webidl.spec.whatwg.org/#buffersource-underlying-buffer instead of consulting [[ViewedArrayBuffer]]

Reply:
Fixed by the latest commit.